### DISCUSSION TITLE
BCI στην εκπαίδευση.

### DISCUSSION THREAD

**Discussion by Pagiatis-Spiros** [sdy60/syllabus#162] [Show and tell]
Σχετικά με αυτό που αναφέρθηκα στην ΟΣΣ, δηλ. το πως χρησιμοποιείται το ΑΙ σαν εργαλείο για την εκπαίδευση.

https://www.youtube.com/watch?v=JMLsHI8aV0g

Το βίντεάκι είναι αρκετά παλιό και σε κάποιες περιπτώσεις γίνεται κατάχρηση της ιδιωτικότητας/ελευθερίας των παιδιών (σε χώρες όπως η Κίνα όμως δεν έπαιζε και πρωτεύοντα ρόλο αυτό).
Μπορούμε όμως με κανόνες και νόμους, να εξάγουμε αρκετά θετικά από το παραπάνω.

**Comment by epidrome** [comment]
Ευχαριστούμε για τον σύνδεσμο, άλλαξα μόνο την κατηγορία της συζήτησης

Περισσότερο για σύστημα BCI μοιάζει παρά με ΑΙ, επίσης μοιάζει με σύστημα Quantified Self (QS)

μάλλον εννοείς ότι τα δεδομένα στο μέλλον θα τροφοδοτήσουν κάποιο σύστημα ΑΙ

Ενδεχομένως να υπάρχει κάποια θετική χρήση των δεδομένων, αλλά γενικά είμαι πολύ επιφυλακτικός,

Το όλο σύστημα είναι πολύ τεχνοκεντρικό και κάνει πάρα πολλές παραδοχές, πχ. Η έλλειψη προσοχής και ο σχεδιασμός της τάξης είναι περισσότερο αξιακά συστήματα παρά παγκόσμιες σταθερές.

Την δεκαετία του 1970 ο Seymour Papert έκανε μια πολύ τίμια προσπάθεια να βάλει την τεχνολογία στις τάξεις με τρόπο ανθρωποκεντρικό: https://pibook.epidro.me/gallery/logo-robot/

**Reply by Pagiatis-Spiros** [reply]
Ευχαριστώ για την αλλαγή της θεματικής κατηγορίας (σκεφτόμουν να το βάλω εκεί αλλά τελικά δεν το έκανα).

Ναι BCI είναι, στην συζήτηση που είχαμε εννοούσα την διάδραση HCI (ότι δηλαδή εμείς έχουμε σαν HCI το ποντίκι, πληκτρολόγιο, οθόνη κτλ..  αλλά πιθανόν μελλοντικά θα αλλάξουν εντελώς τα πράγματα με το BCI).

**_Τα παρακάτω τα αναφέρων χωρίς να έχω κάποια εμπειρία ή γνώση αυτών, οπότε πιθανόν να είναι και λάθος._**
Νομίζω πως τα τελευταία χρόνια τα BCI δουλεύουν αποκλειστικά με ΑΙ και αυτό γιατί η νευροεπιστήμη είναι ακόμη σε νηπιακό επίπεδο στο να αποκωδικοποίηση  σε καλό βαθμό τα εγκεφαλικά σήματα και να τα μετατρέψει σε δεδομένα φιλικά προς εμάς.
Οπότε χρησιμοποιούν EEG sensors και μηχανική μάθηση στα νευρωνικά δίκτυα για να ανακαλύψει η ΑΙ τα μοτίβα των ασθενών ηλεκτρομαγνητικών κυμάτων που παράγει ο εγκέφαλος, για κάθε σκέψη, γράμμα, λέξη, πρόταση, κίνηση κτλ.

Ένα άρθρο που αναφέρει κάποια πράγματα πάνω σε αυτό το τομέα είναι το παρακάτω:
https://iopscience.iop.org/article/10.1088/1741-2552/aab2f2/pdf

Στα αρχικά στάδια το BCI δούλευε σαν ένα απλό ηλεκτρόδιο (χωρίς ΑΙ) το οποίο αντιλαμβανόταν τις ηλεκτρομαγνητικές διαφορές στα νεύρα μας, οι οποίες δημιουργούνταν μέσω των μυϊκών συσπάσεων από τις διάφορες κινήσεις μας.
Ένα τέτοιο παράδειγμα με EOG-BCI είχαμε να αναλύσουμε σε ένα μέρος της 1ης εργασίας στην ΠΛΗ50 πέρυσι, το οποίο χρησιμοποιούσε αισθητήρες EOG στα γυαλιά και μπορούσε να καταλάβει πότε είχαμε την μέγιστη συγκέντρωση μας (έτσι ώστε εκείνη την στιγμή να κάναμε τα απαιτητικά/δύσκολα tasks).
Δείτε στην σελ. 6 (υποχρεωτικά χρησιμοποίησα και το scihub :), δεν το βρήκα free)
https://sci-hub.se/10.1109/MPRV.2020.2994914
